docs/content/docs/start/react-tutorial.md

```diff
@@ -0,0 +1,295 @@
+
+    ```dotnetcli
+    npx create-react-app fluid-react-tutorial --use-npm
+    ```
+
+1. The project is created in a subfolder named `fluid-react-tutorial`. Navigate to it with the command `cd fluid-react-tutorial`.
+1. The project uses two Fluid libraries:
+
+    |Library |Description |
+    |---|---|
+    |fluid&#x2011;experimental/fluid&#x2011;framework |Contains the SharedMap [DataObject](https://fluidframework.com/docs/glossary/#dataobject) that synchronizes data across clients. _This object will hold the most recent timestamp update made by any client._|
+    |fluid&#x2011;experimental/frs&#x2011;client&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;   |Defines the connection to a Fluid Relay Service (FRS) server and defines the starting schema for the [container](https://fluidframework.com/docs/concepts/containers-runtime/). In this tutorial, the FRS server will run on localhost.|
+    &nbsp;
+
+    Run the following command to install the libraries.
+
+    ```dotnetcli
+    npm install @fluid-experimental/frs-client @fluid-experimental/fluid-framework
+    ```
+
+## Code the project
+
+1. Open the file `\src\App.js` in your code editor. Delete all the default `import` statements except the one that imports `App.css`. Then delete all the markup from the `return` statement. The file should look like the following:
+
+    ```js
+    import './App.css';
+    
+    function App() {
+      return (
+  
+      );
+    }
+    
+    export default App;
+    ```
+
+1. Add the following `import` statements:
+
+    ```js
+    import React from "react";
+    import { FrsClient, InsecureTokenProvider } from "@fluid-experimental/frs-client";;
+    import { SharedMap } from "@fluid-experimental/fluid-framework";
+    ```
+
+### Create a container ID helper function
+
+Add the following helper function to the file below the `import` statements. Note the following about this code:
+
+- Every [container](https://fluidframework.com/docs/glossary/#container) must have a unique ID. For the ID, this application will use a truncated version of the UNIX epoch time when the container is first created.
+- The ID is stored in the `window.location.hash` property.
+- The function is called in a useEffect hook that you create in a later step, so it is called every time the application (re)renders.
+
+```js
+const getContainerId = () => {
+    let isNew = false;
+    if (window.location.hash.length === 0) {
+        isNew = true;
+        window.location.hash = Date.now().toString();
+    }
+    const containerId = window.location.hash.substring(1);
+    return { containerId, isNew };
+};
+```
+
+### Configure the FRS client
+
+Add the following constant below the helper function. This object configures the FRS client to connect with an FRS server that runs on localhost. Note that in a production application, you would use a real Secure Token Service to protect access to the FRS server, but during development you can use the dummy service `InsecureTokenProvider`.
```

Comment:
I don't know if "Secure Token Service" is a proper noun in this context or not.

Reply:
```suggestion
Add the following constant below the helper function. This object configures the FRS client to connect with an FRS server that runs on localhost. Note that in a production application, you would use a real security token service to protect access to the FRS server, but during development you can use the dummy service `InsecureTokenProvider`.
```